### PR TITLE
refactor: address TODO comment

### PR DESCRIPTION
**Pull request by PavelKirvelTextkernel**
Link [task](https://textkernel.atlassian.net/browse/ONEUI-411)

### CODE REVIEW

File: src/components/Themeroller/ThemeTuner/__tests__/ThemeTuner.spec.tsx [hunk 1]
```diff
@@ -88,21 +87,60 @@ describe('ThemeTuner component', () => {
...
+        //         target: {
+        //             value: '#00000',
+        //         },
+        //     });
+        //
+        const ch = screen.getAllByRole('log')[1];
+        await ch.focus();
+        // await ch.click();
+        await user.click(ch);
+        await user.type(ch, '#ffffff');
```
Comment (PavelKirvelTextkernel) [inline src/components/Themeroller/ThemeTuner/__tests__/ThemeTuner.spec.tsx:140]: After get input field just hard-code role for it. But input value doesn't change

File: src/components/Themeroller/ThemeTuner/__tests__/ThemeTuner.spec.tsx [hunk 2]
```diff
@@ -88,21 +89,16 @@ describe('ThemeTuner component', () => {
...
 
-        const firstInput = screen.getByRole('spinbutton');
-
-        await user.type(firstInput, '#00000');
+        await user.type(screen.getByRole('textbox'), '#ffffff');
 
-        expect(onChangeMock).toHaveBeenCalledWith({
-            '--color-background': '#00000',
-        });
+        expect(onChangeMock).toHaveBeenCalled();
```
Comment (mukiienko) [inline src/components/Themeroller/ThemeTuner/__tests__/ThemeTuner.spec.tsx:102]: Can we use `toHaveBeenCalledWith`?

Comment (PavelKirvelTextkernel) [reply src/components/Themeroller/ThemeTuner/__tests__/ThemeTuner.spec.tsx:102]: It shows 
image

Comment (mukiienko) [reply src/components/Themeroller/ThemeTuner/__tests__/ThemeTuner.spec.tsx:102]: hmm, then something is wrong with the test itself.

Comment (PavelKirvelTextkernel) [reply src/components/Themeroller/ThemeTuner/__tests__/ThemeTuner.spec.tsx:102]: I think the test is correct. The main problem is that enzyme allow to set the hole value to input. But when we use RTL type function it types by element here. The same problem we have for SelectBase.spec.tsx it was 
image

And then we replace it to 
image

Comment (PavelKirvelTextkernel) [reply src/components/Themeroller/ThemeTuner/__tests__/ThemeTuner.spec.tsx:102]: The solution is to use fireEvent.change I'll apply it for both tests

**Review by mukiienko — APPROVED** [review]

**Pull request merged**